.github/workflows/nodejs.yml

```diff
@@ -38,3 +38,13 @@ jobs:
             - run: yarn test
 
             - run: yarn build
+    test:
+        needs: build
+        timeout-minutes: 10
+        steps:    
+            - name: Install Playwright
+            run: npx playwright install --with-deps
+            - name: Serve Storybook and run tests
+            run: |
+                yarn storybbok \
```

Comment:
Are we 100% sure we want to go our own way on writing, running and debugging tests vs using the [official test tools setup](https://storybook.js.org/addons/@storybook/test-runner) by storybook? I'm personally not that convinced, but happy to do it this way too if the rest of the team thinks it is required, easier or just enough for our use case.

Reply:
I all for using what is easier, so let's use the official tools if you think that is easier!

Reply:
I think having the benefits of community support and tooling are worth to give it a try. Disclaimer: I've never used it myself, just seen this in use by a sibling team at a previous job and didn't hear many complaints.

[Here's an example](https://mealdrop.vercel.app/storybook/?path=/story/userflows-app--end-to-end) I've just found of some of the tooling you get for writing/debugging tests, if you want to give it a try and see how it feels. And [here](https://github.com/yannbf/mealdrop/tree/main/src/pages) seems to be the source code of how the tests are organized.

One thing I like is that the test code sticks together with the "story", so it is easier to keep together a component's documentation, tests, etc.

Reply:
Playwright tests for iso-editor are going to be more focused on functionality rather than UI/components for now (13901df), but I'm hoping to go the `@storybook/test-runner` route for wp-components! 🔥